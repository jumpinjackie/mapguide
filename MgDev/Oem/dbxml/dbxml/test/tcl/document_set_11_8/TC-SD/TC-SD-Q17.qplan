<XQuery>
  <FLWOR>
    <ForBinding name="ent">
      <QueryPlanToAST>
        <NodePredicateFilterQP uri="" name="#tmp2">
          <StepQP axis="child" name="e" nodeType="element">
            <StepQP axis="child" name="dictionary" nodeType="element">
              <ValueQP index="node-attribute-equality-string" operation="eq" child="id" value="E1"/>
            </StepQP>
          </StepQP>
          <ValueFilterQP comparison="eq" general="true">
            <StepQP axis="attribute" name="id" nodeType="attribute">
              <VariableQP name="#tmp2"/>
            </StepQP>
            <Sequence>
              <AnyAtomicTypeConstructor value="E1" typeuri="http://www.w3.org/2001/XMLSchema" typename="string"/>
            </Sequence>
          </ValueFilterQP>
        </NodePredicateFilterQP>
      </QueryPlanToAST>
    </ForBinding>
    <ForBinding name="related">
      <QueryPlanToAST>
        <NodePredicateFilterQP uri="" name="#tmp5">
          <StepQP axis="child" name="e" nodeType="element">
            <StepQP axis="child" name="dictionary" nodeType="element">
              <ValueQP index="node-attribute-equality-string" operation="eq" child="id">
                <Atomize>
                  <QueryPlanToAST>
                    <StepQP axis="child" name="cr" nodeType="element">
                      <StepQP axis="child" name="et" nodeType="element">
                        <VariableQP name=":ent"/>
                      </StepQP>
                    </StepQP>
                  </QueryPlanToAST>
                </Atomize>
              </ValueQP>
            </StepQP>
          </StepQP>
          <ValueFilterQP comparison="eq" general="true">
            <StepQP axis="attribute" name="id" nodeType="attribute">
              <VariableQP name="#tmp5"/>
            </StepQP>
            <Atomize>
              <QueryPlanToAST>
                <StepQP axis="child" name="cr" nodeType="element">
                  <StepQP axis="child" name="et" nodeType="element">
                    <VariableQP name=":ent"/>
                  </StepQP>
                </StepQP>
              </QueryPlanToAST>
            </Atomize>
          </ValueFilterQP>
        </NodePredicateFilterQP>
      </QueryPlanToAST>
    </ForBinding>
    <DOMConstructor type="element">
      <Name>
        <Sequence>
          <ATQNameConstructor uri="" prefix="" localname="Output" typeuri="http://www.w3.org/2001/XMLSchema" typename="QName"/>
        </Sequence>
      </Name>
      <Children>
        <ContentSequence>
          <QueryPlanToAST>
            <StepQP axis="child" name="hw" nodeType="element">
              <StepQP axis="child" name="hwg" nodeType="element">
                <VariableQP name=":related"/>
              </StepQP>
            </StepQP>
          </QueryPlanToAST>
        </ContentSequence>
      </Children>
    </DOMConstructor>
  </FLWOR>
</XQuery>
