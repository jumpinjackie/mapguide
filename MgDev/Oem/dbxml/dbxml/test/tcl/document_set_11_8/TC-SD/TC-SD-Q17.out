<Output><hw>commixture</hw></Output><Output><hw>bard</hw></Output>